apps/condo/domains/organization/components/ErrorsContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from '@emotion/styled'
+import { useIntl } from '@core/next/intl'
+import { Col } from 'antd'
+import React from 'react'
+import { colors } from '@condo/domains/common/constants/style'
+
+export const ErrorsWrapper = styled.div`
+  display: inline-block;
+  padding: 9px 16px;
+  border-radius: 8px;
+  background-color: ${colors.beautifulBlue[5]};
+`
```

Comment:
It seems, like it also should be extracted into `common` domain.
Create a task for it, please.